app/src/main/java/com/innowisegroup/datetimepicker/DateTimePickerDialog.kt

```diff
@@ -180,7 +207,42 @@ class DateTimePickerDialog(
         this.show(fragmentManager, DIALOG_TAG)
     }
 
-    private companion object {
-        const val DIALOG_TAG = "date_time_picker_dialog"
+    companion object {
+        const val UPDATE_TIME_TAB_TITLE_REQUEST_KEY = "updateTimeTabTitleKey"
+        const val UPDATE_DATE_TAB_TITLE_REQUEST_KEY = "updateDateTabTitleKey"
+
+        const val UPDATE_TIME_TAB_TITLE_KEY = "updateTimeTabTitleKey"
+        const val UPDATE_DATE_TAB_TITLE_KEY = "updateDateTabTitleKey"
+
+        private const val DIALOG_TAG = "date_time_picker_dialog"
+
+        private const val INITIAL_LOCAL_DATE_TIME = "initialLocalDateTime"
+        private const val INITIAL_LOCAL_TIME = "initialLocalTime"
+        private const val MIN_LOCAL_DATE_TIME = "minLocalDateTime"
+        private const val MAX_LOCAL_DATE_TIME = "maxLocalDateTime"
+        private const val WRAP_SELECTION_WHEEL = "wrapSelectionWheel"
+        private const val FRAGMENT_TO_CREATE = "fragmentToCreate"
+
+        @JvmStatic
+        fun newInstance(
+            initialLocalDateTime: LocalDateTime?,
+            initialLocalTime: LocalTime?,
+            minLocalDateTime: LocalDateTime?,
+            maxLocalDateTime: LocalDateTime?,
+            fragmentToCreate: FragmentToCreate,
+            wrapSelectionWheel: Boolean
+        ): DateTimePickerDialog {
+            val args = Bundle().apply {
```

Comment:
let's use bundleOf() function

Reply:
![image](https://user-images.githubusercontent.com/81771801/182867860-506d6c6b-9947-46c7-a6a0-525a6a26f34d.png)

Reply:
Yeah. It comes with separate dependency. Then let's skip it.